AllReadyApp/Web-App/AllReady/Features/Users/GetUserIdQueryHandler.cs

```diff
@@ -6,16 +6,16 @@
 
 namespace AllReady.Features.Users
 {
-    public class GetUserIdCommandHandler : IAsyncRequestHandler<GetUserIdCommand, string>
+    public class GetUserIdQueryHandler : IAsyncRequestHandler<GetUserIdQuery, string>
```

Comment:
Yeah, fair point. I was originally assuming more work needed which I was trying to take away from the action and therefore keeping the controller light, but agree for the one line and dependency it saves, this handler is a bit unnecessary. Will refactor it.